src/OpenTelemetry.Instrumentation.AspNet/.publicApi/net462/PublicAPI.Unshipped.txt

```diff
@@ -6,8 +6,17 @@ OpenTelemetry.Instrumentation.AspNet.AspNetInstrumentationOptions.Filter.get ->
 OpenTelemetry.Instrumentation.AspNet.AspNetInstrumentationOptions.Filter.set -> void
 OpenTelemetry.Instrumentation.AspNet.AspNetInstrumentationOptions.RecordException.get -> bool
 OpenTelemetry.Instrumentation.AspNet.AspNetInstrumentationOptions.RecordException.set -> void
+OpenTelemetry.Instrumentation.AspNet.AspNetMetricsInstrumentationOptions
+OpenTelemetry.Instrumentation.AspNet.AspNetMetricsInstrumentationOptions.AspNetMetricsInstrumentationOptions() -> void
+OpenTelemetry.Instrumentation.AspNet.AspNetMetricsInstrumentationOptions.Enrich.get -> OpenTelemetry.Instrumentation.AspNet.AspNetMetricsInstrumentationOptions.EnrichFunc?
+OpenTelemetry.Instrumentation.AspNet.AspNetMetricsInstrumentationOptions.Enrich.set -> void
+OpenTelemetry.Instrumentation.AspNet.AspNetMetricsInstrumentationOptions.EnrichFunc
+OpenTelemetry.Instrumentation.AspNet.AspNetMetricsInstrumentationOptions.Filter.get -> OpenTelemetry.Instrumentation.AspNet.AspNetMetricsInstrumentationOptions.FilterFunc?
+OpenTelemetry.Instrumentation.AspNet.AspNetMetricsInstrumentationOptions.Filter.set -> void
+OpenTelemetry.Instrumentation.AspNet.AspNetMetricsInstrumentationOptions.FilterFunc
```

Comment:
It is opposed to https://github.com/open-telemetry/opentelemetry-dotnet/pull/4981 for ASP.NET Core metrics.

@vishweshbankwar, I suppose that we should be able to release also ASP.NET as stable together/after ASP.NET. It is potential blocker for this.

Reply:
From what I can gather, these features are for .NET 8 and will not be available for .NET Framework (as per @cijothomas comment) which this library targets? What would be the way forward for enrich filter/functionailty here then?

Reply:
https://github.com/open-telemetry/opentelemetry-dotnet/pull/4981 does not apply here. Regarding the stable release of ASP.NET, there is no ETA for it.